clans/src/main/resources/configs/config.yml

```diff
@@ -160,28 +169,28 @@ command:
       requiredRank: ADMIN
     bank:
       enabled: true
-      requiredRank: ADMIN
+      requiredRank: PLAYER
       withdraw:
         enabled: true
-        requiredRank: ADMIN
+        requiredRank: PLAYER
       deposit:
         enabled: true
-        requiredRank: ADMIN
+        requiredRank: PLAYER
```

Comment:
currently still admin only, only purpose for the bank is to store gold to collect from sold auctions